### ISSUE TITLE
Verify US postalcode record hierarchies 

### ISSUE THREAD

**Issue by stepps00** [whosonfirst-data/whosonfirst-data#794]
When bundling all `postalcode` records in California ([link](https://whosonfirst.mapzen.com/spelunker/download/85688637/?exclude=nullisland#10/37.7691/-122.4316)), some records are not added to the bundle, even though they include a correct hierarchy (see below):

**Example:**

data/554/784/701/554784701.geojson

```
    "wof:hierarchy":[
        {
            "continent_id":102191575,
            "country_id":85633793,
            "county_id":102087579,
            "locality_id":85922583,
            "postalcode_id":554784701,
            "region_id":85688637
        }
    ],
```
The built hierarchy in the Spelunker, though, does not include California (see below). 
 
<img width="272" alt="screen shot 2017-05-19 at 9 39 56 am" src="https://cloud.githubusercontent.com/assets/18567700/26257887/27ca4324-3c77-11e7-854a-d9e4e026f218.png">

This is due to S3 records being out of sync with GitHub - GitHub records do not include California, while the S3 records do. Sync the records and verify that all `postalcode` records in the US include a region in their hierarchy.

**Comment by stepps00** [comment]
The `postalcode` records were updated and in sync across S3 and GitHub on Aug 8, 2016. 

Then, commit https://github.com/whosonfirst-data/whosonfirst-data-postalcode-us/commit/75aeaca2b056aa29d7b20cdab213654ba8b26d58 updated the hierarchies of the `postalcode` records in GitHub, which removed the `region_id` from these feature's hierarchies.

Since that commit only removed California from the hierarchy of these features, I think the proper thing to do here is to revert postalcodes in GitHub back to their state as of Aug 8, 2016 (pre-commit).

**Comment by nvkelso** [comment]
Were any other postcodes affected (outside of SF)?

Would reverting also fix the postcodes in CA but outside of SF?

**Comment by stepps00** [comment]
Yes, this would affect any `postalcode` record updated in that commit.

Looking through [all the commits in the `postalcode-us` repo](https://github.com/whosonfirst-data/whosonfirst-data-postalcode-us/commits/master), the only other updates were to Barrow/ Utqiaġvik, Alaska, so reverting back to the last commit for all of CA shouldn't cause any additional updates to be lost.
